### PR TITLE
Update age.ts

### PR DESCRIPTION
**Pull request by gsbelarus**

### CODE REVIEW

File: src/age.ts [hunk 1]
```diff
@@ -1,3 +1,7 @@
 export const getAgeString = (years: number) => {
-  return `The age of the person: ${years} years.`;
+  if (age > 0 && age < 180) {
+    return `The age of the person: ${years} years.`;
+  } else {
+    return 'Invalid age!';
+  }
 }
```
Comment (gsbelarus) [inline src/age.ts:7]: The code patch modifies the `getAgeString` function to include validation for age before returning the formatted string. Here's a brief review:

1. **Parameter vs Variable Name Mismatch**:
   There is an inconsistency with the variable names. The function parameter is named `years`, but within the `if` condition, it uses `age`. This would result in a reference error because `age` is not defined anywhere in the scope of the function. To fix this, you should refer to the `years` parameter instead.

2. **Boundary Conditions**:
   The range check for age is between greater than 0 and less than 180. This implies the code doesn't consider 0 or 180 as valid ages. If these should be included as valid, the conditions need to be non-strict (i.e., `>= 0` and `<= 180`).

3. **Code Commenting**:
   Adding comments for the validation might help other developers understand why the range was chosen (`> 0 && < 180`). For example, it could be helpful to know that you're assuming 180 to be the upper limit of human age based on current records.

4. **Return Type Consistency**:
   Both code paths return a string. This consistency is good; make sure that this matches what the consumers of the function expect.

5. **Unit Testing**:
   It would be beneficial if there were associated unit tests for this function to test edge cases such as ages -1, 0, 179, 180, and over 180 to ensure the function behaves as expected.

6. **Internationalization and Language**:
   The function output is hardcoded to produce text in English, which may not be suitable for applications supporting multiple languages. It's more of a design consideration than a bug issue—consider whether this function should be internationalized.

Here is the corrected function:

```javascript
export const getAgeString = (years: number) => {
  if (years >= 0 && years <= 180) { // Note the change from > to >= and < to <= to include 0 and 180
    return `The age of the person: ${years} years.`;
  } else {
    return 'Invalid age!';
  }
};
```

In summary, fix the variable name mismatch and evaluate boundary conditions & assumptions. Consider the wider context the function will be used in, like internationalization requirements or how it integrates with other parts of the system, especially regarding error handling practices.
